srcpkgs/i2pd/template

```diff
@@ -24,7 +24,7 @@ make_dirs="/var/lib/i2pd 0700 _i2pd _i2pd"
 
 case "${XBPS_TARGET_MACHINE}" in
 	x86_64*|i686*) ;;
-	*) make_build_args+=" USE_AESNI=no USE_AVX=no" ;;
+	*) make_build_args+=" USE_AVX=no" ;;
```

Comment:
I think it should be:
```sh
case "${XBPS_TARGET_MACHINE}" in
	x86_64*) ;;
	*) make_build_args+=" USE_AESNI=no USE_AVX=no" ;;
esac
```

Reply:
Not sure what to do with this. So I closed the pull request